frontend/src/pages/quiz/tasks/[[...filters]].tsx

```diff
@@ -66,6 +63,18 @@ const Tasks: FC<TasksProps> = ({ applyingBenefits, beforeRetiring, filters, rece
               {t('restart-quiz')}
             </Button>
           </div>
+          <p>Answers:</p>
```

Comment:
I'm confused by section (answers).  It doesn't appear in the Figma mockups.  Is it a placeholder?

Reply:
It's temporary for debugging until you implement the left navigation section :)